commands/channel_test.go

```diff
@@ -129,3 +129,150 @@ func (s *MmctlUnitTestSuite) TestSearchChannelCmdF() {
 		s.EqualError(err, "Team "+teamArg+" was not found")
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestCreateChannelCmdF(){
+	mockChannelName := "Mock Channel"
+	mockChannelDisplayName := "Mock Channel Display Name"
+	mockTeam := &model.Team{Name: "team", DisplayName: "team"}
+
+	s.Run("Create channel with no name returns an error", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		err := createChannelCmdF(s.client, cmd, []string{})
+
+		s.Require().Equal(err.Error(), "Name is required")
+		s.Require().Len(printer.GetLines(), 0)
+
+	})
+
+	s.Run("Create channel with a name but no Display Name returns an error", func() {
+		printer.Clean()
+
+		cmd := &cobra.Command{}
+		cmd.Flags().String("name", mockChannelName, "")
+		err := createChannelCmdF(s.client, cmd, []string{})
+
+		s.Require().Equal(err.Error(),"Display Name is required")
+		s.Require().Len(printer.GetLines(), 0)
+	})
+
+	s.Run("Create channel with a Name but no Display Name returns an error", func() {
```

Comment:
This description is wrong, here we are creating a channel with Displa Name bot no Name